CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.0)
+project(iotools
+	VERSION 1.8
+	LANGUAGES C)
+
+if (NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE Debug)
+endif()
+
+option(STATIC "Build statically-linked iotools?" ON)
+
+# Override implicit "-g"
+set(CMAKE_C_FLAGS_DEBUG "-O0 -ggdb")
```

Comment:
Hmm... Have you tested that it still works as expected? `-O0` is a beast and sometimes may lead to unusable code generation.

Reply:
Fair enough.